packages/helpers/src/cms/getCmsTranslate.ts

```diff
@@ -7,7 +7,7 @@
  */
 export function getCmsTranslate(
   key: string,
-  params?: { [key: string]: string | number | null } | null,
+  params?: { [key: string]: string | number | null | undefined } | null,
```

Comment:
same with the undefined definition
```suggestion
  params?: { [key: string]?: string | number | null } | null,
```